src/Microsoft.Health.Dicom.Core/Features/CustomTag/AddCustomTagService.cs

```diff
@@ -3,6 +3,7 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System;
```

Comment:
Is this necessary?

Reply:
I think you find a bug in Visual Studio -- it is indicated as used, but deleting that line doesn't fail any thing.
![image](https://user-images.githubusercontent.com/67345894/107402853-4ab80500-6ab9-11eb-9176-01ae18b4451e.png)